### PR TITLE
feat(toolbar): add background, center align elements & remove checkmark

### PR DESCRIPTION
**Pull request by akshatakulkarni25**
Signed-off-by: Akshata Kulkarni <akshatakulkarni2509@gmail.com>
Change-Id: Id44ba88a8645ff4bcb29cc645b180693c7dcece9

* Resolves: #  
* #2072 
* Target version: master 

### Summary
Resolves the issue by adding a conditional check for item that is selected and changing its background colour, text alignment and removes checked icon. 

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [x] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [x] Documentation (manuals or wiki) has been updated or is not required

Screenshot 2022-11-17 at 11 22 55 PM

### CODE REVIEW

**Comment by Rash419** [comment]
@akshatakulkarni25 Thanks for the PR , I will review it : )

File: browser/css/w2ui-1.5.rc1.css [hunk 1]
```diff
@@ -481,6 +481,8 @@ input[type=checkbox].w2ui-toggle:focus {
 .w2ui-overlay table.w2ui-drop-menu td.menu-text {
   padding: 8px 10px 8px 5px;
   width: auto;
+  text-align: center;
+  font-size: 16px;
```
Comment (pedropintosilva) [inline browser/css/w2ui-1.5.rc1.css:485]: Please these additional rules at the end of the `toolbar.css` file. This way we keep the original w2ui file clean and less prone to changes and use the later file to overwrite those rules.

Also maybe we could use CSS var for the font-size, you can add a new one (example `--toolbar-drop-menu-font-size`) in the top of `color-palette.css`

File: browser/css/w2ui-1.5.rc1.css [hunk 2]
```diff
@@ -528,6 +530,9 @@ input[type=checkbox].w2ui-toggle:focus {
 /*************************************************
 * ---- Common Classes ----
 */
+.mitem-selected {
+  background-color: #d3d3d3;
+}
```
Comment (pedropintosilva) [inline browser/css/w2ui-1.5.rc1.css:535]: Please add these at the end of  `toolbar.css` instead

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/css/w2ui-1.5.rc1.css (hunk 2 above)
Comment (pedropintosilva) [reply browser/css/w2ui-1.5.rc1.css:535]: and consider using css var. maybe `--color-overlay` ?

**Comment by akshatakulkarni25** [comment]
@pedropintosilva I have completed the review changes please verify.

File: browser/css/toolbar.css [hunk 3]
```diff
@@ -1289,3 +1289,11 @@ html[dir='rtl'] #userListPopover::after {
 	font-size: var(--default-font-size);
 	color: var(--color-main-text);
 }
+.w2ui-overlay table.w2ui-drop-menu td.menu-text{
+	text-align: center;
+  	font-size: var(--toolbar-drop-menu-font-size);
```
Comment (pedropintosilva) [inline browser/css/toolbar.css:1294]: Please remove the space before the tab in this line

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/css/toolbar.css (hunk 3 above)
Comment (pedropintosilva) [reply browser/css/toolbar.css:1294]: I'm asking these because otherwise the lint test fails and it's impossible to build it

**Comment by pedropintosilva** [comment]
Also, and after the about request is done would be nice to have your two commits in one, so to avoid having a "fix of a fix" in the git history. For these you need to squash the commits. For that you need to type in your terminal:

```bash
git rebase -i HEAD~2
```

Editor will open
```
pick e2ecaadaf feat(toolbar): add background, center align elements & remove checkmark
pick b61257b8c fix(code review): created css var for color overlay & toolbar dropdown font size
```

Change the first word of the last line to `squash`, save and close the text editor. A new text editor window will open with:

```
# This is a combination of 2 commits.
# This is the 1st commit message:

feat(toolbar): add background, center align elements & remove checkmark

Signed-off-by: Akshata Kulkarni <akshatakulkarni2509@gmail.com>
Change-Id: Id44ba88a8645ff4bcb29cc645b180693c7dcece9

# This is the commit message #2:

fix(code review): created css var for color overlay & toolbar dropdown font size

# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# Author:    Akshata Kulkarni <akshatakulkarni2509@gmail.com>
# Date:      Thu Nov 17 18:29:40 2022 +0000
#
# interactive rebase in progress; onto 510f5bb7a
# Last commands done (2 commands done):
#    pick e2ecaadaf feat(toolbar): add background, center align elements & remove checkmark
#    squash b61257b8c fix(code review): created css var for color overlay & toolbar dropdown font size
# No commands remaining.
# You are currently rebasing branch 'toolbar-style-change' on '510f5bb7a'.
#
# Changes to be committed:
#	modified:   browser/css/color-palette.css
#	modified:   browser/css/toolbar.css
#	modified:   browser/js/w2ui-1.5.rc1.js
#

```

edit that text and remove everything after the first commit message so, this would be the end result:
```
feat(toolbar): add background, center align elements & remove checkmark

Signed-off-by: Akshata Kulkarni <akshatakulkarni2509@gmail.com>
Change-Id: Id44ba88a8645ff4bcb29cc645b180693c7dcece9


```

**Review by eszkadev — CHANGES_REQUESTED** [review]
we are removing w2ui: https://github.com/CollaboraOnline/online/pull/8682
this will no longer not apply to the code, could you check the new PR, if it is fixed or needs something to do?

**Pull request closed without merging**